cpp/arcticdb/pipeline/slicing.hpp

```diff
@@ -67,9 +67,9 @@ SlicingPolicy get_slicing_policy(
 
 std::vector<FrameSlice> slice(InputTensorFrame &frame, const SlicingPolicy& slicer);
 
-std::vector<folly::Future<SliceAndKey>> write_slices(
+folly::Future<std::vector<SliceAndKey>> write_slices(
     const InputTensorFrame &frame,
-    const std::vector<FrameSlice> &slices,
+    std::vector<FrameSlice> slices,
```

Comment:
I'm confused by this. Is this meant to be the function prototype of `write_slices` from `write_frame.cpp`? Because the signatures don't match so it can't be.

Reply:
Modifying this file was just a silly mistake I did, which was not detected by the compiler because no function is implemented/used for this header. So, I removed it completely. The one I am using relates to the implementation in write_slices. Also, changed the header of write_slices to be consistent